sally/__manifest__.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "Sally's Flower Shop",
+    'summary': """Sally's Flower Shop""",
+    'description': """
+        This is a practice project for odoo course
+    """,
+    'author': "Hadi Jafari",
+    'website': "https://haadijafari.ir",
+    'category': 'Flower Shop',
+    'version': '1.0.0',
+    'depends': [
+        'base',
+        'product',
+        'sale',
+        'sale_management',
+        'stock',
+        'website',
+        'website_sale',
+        'base_geolocalize',
```

Comment:
TIP: you don't need to import base, sale, sale_management, stock, website as they are  dependencies in website_sale and sale_stock